src/EditorFeatures/Core/Implementation/Debugging/IBreakpointResolutionService.cs

```diff
@@ -9,10 +9,11 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Debugging
 {
+    // Used by TypeScript.
```

Comment:
Not sure we want this, should it ever change. We're just bringing up better tooling to track this instead.